### ISSUE TITLE
Language notification

### ISSUE THREAD

**Issue by BrettMayson** [IDI-Systems/acre2#657]
With Babel there is no way to know what language someone is speaking. In reality you can easily tell what language someone is speaking. This is also an issue when someone else switches between languages that the player knows. The player will have no idea that they have switched languages.

I think there should be a little notification saying "Brett is speaking German" when someone either switches languages or you encounter someone for the first time not speaking your default language.

**Comment by TheMagnetar** [comment]
>In reality you can easily tell what language someone is speaking.

Not really. I cannot tell the diffwrencworld between Chinese or Vietnamese.

I agree however that you need a notification for switching to "known" languages.

**Comment by BrettMayson** [comment]
I mean, most people can at least tell between a southeast Asia, eastern European, or a romance language for example. There just isn't any regions in ACRE. It shouldn't be a big issue though, the notification for known languages will be more important.
